app/src/main/res/layout/listview_cell_search_results.xml

```diff
@@ -33,14 +33,14 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginEnd="16sp"
-        android:width="108sp"
+        android:width="120sp"
         android:background="@android:color/darker_gray"
         android:gravity="center"
         android:paddingBottom="12sp"
         android:paddingTop="12sp"
         android:textAppearance="@style/text.large"
         android:textColor="@color/white"
-        tools:text="Kh.1234"/>
+        tools:text="BOK/1234"/>
```

Comment:
This string is just for us to see when we look at a preview rendering of the UI in Android Studio.  We just need to provide some sort of example string there, so that the UI elements have approximately the right size to lay them out.